src/services/paras/ParasService.ts

```diff
@@ -45,51 +71,44 @@ export class ParasService extends AbstractService {
 		return {
 			at,
 			fundInfo,
+			// TOOD would it make snese to merge the below derived info into `fundInfo`
```

Comment:
Little confused on  this could you elaborate?

Reply:
Basically should we add `leasePeriods` as a key within the `fundInfo` object?